### DISCUSSION TITLE
Getting "Error 111: Connection Refused" when trying to connect to terminal

### DISCUSSION THREAD

**Discussion by nwill111** [AlexProgrammerDE/balena-minecraft-server#157] [Q&A]
When I try to access the server terminal by connecting to balena-rcon I get a connection refused error. Does anyone have any idea why this could be?

**Comment by viedok** [comment]
You can't connect via the web dashboard to the pi, you will need a [RCON client](https://sourceforge.net/projects/mcrcon/) if you want to use the command line or a [FTP client](https://filezilla-project.org) if you want to modify the files of the server. Alternatively, you can use plugins to communicate with the server. I personally use [DiscordSRV](https://www.spigotmc.org/resources/discordsrv.18494/) for relaying the chat to my discord server and it also lets me use operator commands in a separate private channel.
